tests/load/locustfiles/locustfile.py

```diff
@@ -32,6 +32,9 @@
 from pydantic import ValidationError
 from websocket import WebSocketApp, WebSocketConnectionClosedException
 
+Message: TypeAlias = HelloMessage | NotificationMessage | RegisterMessage | None
+Record: TypeAlias = HelloRecord | NotificationRecord | RegisterRecord | None
```

Comment:
The [doc](https://docs.python.org/3/library/typing.html#type-aliases) can explain better than me. 

Here I'm declaring a type alias so that I don't have to write `HelloMessage | NotificationMessage | RegisterMessage | None` each time, instead I can just say `Message`.